tests/test_examples.py

```diff
@@ -16,26 +16,32 @@
 
 from gordo_components.machine.dataset.data_provider.providers import DataLakeProvider
 from gordo_components.machine.dataset.datasets import TimeSeriesDataset
+from gordo_components.machine.dataset.sensor_tag import SensorTag
 
 logger = logging.getLogger(__name__)
 
+CONFIG = dict(
+    train_start_date=dateutil.parser.isoparse("2014-07-01T00:10:00+00:00"),
+    train_end_date=dateutil.parser.isoparse("2015-01-01T00:00:00+00:00"),
+    tag_list=[SensorTag(name=f"Tag {i}", asset=None) for i in range(10)],
+)
```

Comment:
Do you think this should be a fixture? :man_shrugging:

Reply:
I did it as a global since it is used in a helper function that is passed as the return value to a `mock.patch`.  I don't THINK a fixture could be used here.

Reply:
Just tried trying this out in a MWE, If I were to make that helper function a fixture, then use that as a return value I get this:
```
Fixture "fn" called directly. Fixtures are not meant to be called directly,
but are created automatically when test functions request them as parameters.
See https://docs.pytest.org/en/latest/fixture.html for more information about fixtures, and
https://docs.pytest.org/en/latest/deprecations.html#calling-fixtures-directly about how to update your code.
```